spec/support/geocoder_stubs.rb

```diff
@@ -39,3 +39,13 @@
     },
   ]
 )
+
+Geocoder::Lookup::Test.add_stub(
```

Comment:
What motivated this change?

Reply:
I don't remember. I think there was a test that was acting up without it 🤔

Reply:
I'm gonna yank this and see what happens

Reply:
Just pushed that change